--- a/recipes/net-snmp/net-snmp_5.7.1.bb
+++ b/recipes/net-snmp/net-snmp_5.7.1.bb
@@ -1,10 +1,11 @@
 require net-snmp.inc
-PR = "${INC_PR}.0"
+PR = "${INC_PR}.1"
 LIC_FILES_CHKSUM = "file://README;beginline=3;endline=8;md5=7f7f00ba639ac8e8deb5a622ea24634e"
 
 DEFAULT_PREFERENCE = "1"
 
 SRC_URI = "${SOURCEFORGE_MIRROR}/net-snmp/net-snmp-${PV}.tar.gz \
+        file://net-snmp-libnl.patch \
         file://init \
         file://snmpd.conf \
         file://snmptrapd.conf"
 
